### DISCUSSION TITLE
Exporting RabbitMQ metrics to AWS Cloudwatch

### DISCUSSION THREAD

**Discussion by ahmad-mujtaba09** [rabbitmq/rabbitmq-server#11032] [Questions] [answered]
can any1 guide me to export rabbitmq metrics to cloudwatch i am using rabbitmq 3.7.9 erlang 21.1
i hv followed this link https://github.com/noxdafox/rabbitmq-cloudwatch-exporter
the problem is that uptil release 0.3.1 im getting the following error
Apr 18 15:03:34 fedora rabbitmq-server[193136]:  completed with 4 plugins.
Apr 18 15:04:33 fedora rabbitmq-server[193136]: 15:04:33.918 [error] GenServer :rabbitmq_cloudwatch_exporter terminating
Apr 18 15:04:33 fedora rabbitmq-server[193136]: ** (BadFunctionError) expected a function, got: nil
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (rabbitmq_cloudwatch_exporter) lib/rabbitmq_cloudwatch_exporter/exporter.ex:56: anonymous fn/2 in RabbitMQCloudWatchExporter.Exporter.handle_info/2
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (elixir) lib/enum.ex:2986: Enum.flat_map_list/2
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (rabbitmq_cloudwatch_exporter) lib/rabbitmq_cloudwatch_exporter/exporter.ex:56: RabbitMQCloudWatchExporter.Exporter.handle_info/2
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (stdlib) gen_server.erl:637: :gen_server.try_dispatch/4
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (stdlib) gen_server.erl:711: :gen_server.handle_msg/6
Apr 18 15:04:33 fedora rabbitmq-server[193136]:     (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Apr 18 15:04:33 fedora rabbitmq-server[193136]: Last message: :export_metrics
and the newer versions are not compatible with rabbitmq 3.7.9 erlang 21.1
can any1 guide me regarding the same
or provide any other soln to send rabbitmq metrics to aws cloudwatch

**Comment by lukebakken** [comment, marked as answer]
RabbitMQ 3.7.9 is completely out of support. We can't assist you unless you use a supported version of RabbitMQ and Erlang - https://www.rabbitmq.com/release-information

**Reply by michaelklishin** [reply]
In fact, even then we can't since we do not maintain that plugin.
